### DISCUSSION TITLE
There is a bug that made Minecraft log spam

### DISCUSSION THREAD

**Discussion by maxplayz109X** [FabricMC/fabric-api#2859] [Player support]
When I use config mod, Minecraft will log spam for no reason people say is a fabric fault https://user-images.githubusercontent.com/123273371/213888465-451c38bf-43c9-4965-838f-ef41233761a3.MOV I talked to the owner of the config mod, they say is not config fault and people said it is your mod fault, pls fix this bug

**Comment by maxplayz109X** [comment]
Here is the clear video https://youtu.be/Wz6wANe_pbo

**Comment by warjort** [comment]
That looping behaviour is usually caused by a mod breaking the resource pack loading.

You need to post a link to your logs/latest.log so we can see the error.
